cel.md

```diff
@@ -0,0 +1,13 @@
+## Setup
+
+For more details about the CEL input settings, check the [Filebeat documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-cel.html).
+
+### Collecting logs from CEL
+
+To collect logs via CEL, select **Collect logs via CEL** and configure the following parameters:
+
+- Dataset name: Dataset to write data to. Changing the dataset will send the data to a different index. You can't use `-` in the name of a dataset and only valid characters for [Elasticsearch index names](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html).
```

Comment:
The link redirects https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html to https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-create. Maybe just update it to latter?